### DISCUSSION TITLE
Tailwind CSS: Cannot read property 'modifier' of undefined

### DISCUSSION THREAD

**Discussion by taeyoung1012** [tailwindlabs/tailwindcss#9670] [Help]
![image](https://user-images.githubusercontent.com/102844875/198218904-688cd700-a3b0-4eb7-a2f2-e0146691f169.png)

errors: The style has taken effect, but the ide does not have any intellisense and reports an error

`Found Tailwind CSS config file: /Users/imtaeyeong/Dev/pocket/tailwind.config.js
Loaded postcss v8.4.18: /Users/imtaeyeong/Dev/pocket/node_modules/postcss
Loaded tailwindcss v3.2.1: /Users/imtaeyeong/Dev/pocket/node_modules/tailwindcss
[Error - 4:26:30 PM] Tailwind CSS: Cannot read property 'modifier' of undefined
TypeError: Cannot read property 'modifier' of undefined
    at api.addVariant (/Users/imtaeyeong/Dev/pocket/node_modules/tailwindcss/lib/lib/setupContextUtils.js:578:40)
    at /Users/imtaeyeong/Dev/pocket/node_modules/tailwindcss/lib/lib/setupContextUtils.js:538:38
    at /Users/imtaeyeong/.vscode/extensions/bradlc.vscode-tailwindcss-0.7.7/dist/server/tailwindServer.js:56:364097
    at Array.forEach (<anonymous>)
    at /Users/imtaeyeong/.vscode/extensions/bradlc.vscode-tailwindcss-0.7.7/dist/server/tailwindServer.js:56:363587
    at /Users/imtaeyeong/.vscode/extensions/bradlc.vscode-tailwindcss-0.7.7/dist/server/tailwindServer.js:56:365792
    at Generator.next (<anonymous>)
    at s (/Users/imtaeyeong/.vscode/extensions/bradlc.vscode-tailwindcss-0.7.7/dist/server/tailwindServer.js:56:344167)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
`

**Comment by wongjn** [comment]
Try updating the Tailwind Intellisense extension to v0.9.0 or higher

**Reply by taeyoung1012** [reply]
Thank you!!!

**Reply by wongjn** [reply]
Try updating the Tailwind Intellisense extension to v0.9.0 or higher
